### ISSUE TITLE
npm i -g grunt-generate-configs fail

### ISSUE THREAD

**Issue by marocas** [creynders/grunt-generate-configs#10]
```
Microsoft Windows [Versão 6.1.7601]
Copyright (c) 2009 Microsoft Corporation. Todos os direitos reservados.

C:\Users\rAs00l>npm install grunt-generate-configs -g
/


> js2coffee@0.2.7 preinstall C:\Users\rAs00l\AppData\Roaming\npm\node_modules\gr
unt-generate-configs\node_modules\js2coffee
> npm run-script ./cyclic.js

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "run-script" "./cyclic.js"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1

npm ERR! missing script: ./cyclic.js
npm ERR!
ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\rAs00l\AppData\Roaming\npm\node_modules\grunt-generate-con
figs\node_modules\js2coffee\npm-debug.log
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "install" "grunt-generate-configs" "-g"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! js2coffee@0.2.7 preinstall: `npm run-script ./cyclic.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the js2coffee@0.2.7 preinstall script 'npm run-script ./cycli
c.js'.
npm ERR! This is most likely a problem with the js2coffee package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run-script ./cyclic.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls js2coffee
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\rAs00l\npm-debug.log
```

Hello, i was wondering if you still working on this project.
If you need the log file, tell me, so i can upload it for you.
Best regards.

**Comment by creynders** [comment]
As the error report says, it seems to be a problem with js2coffee. I tried updating to the last version but unfortunately it explodes all over the place. If it gets fixed I'll update grunt-generate-configs dependency on it.

**Comment by arminrosu** [comment]
For anyone else stumbling over this issue. I've clone this repo and updated `js2coffee` requirement to `2.x.x`. `js2coffee` has coffee-script as a devDependency but is actually a proper dependency and will still will not work.

A workaround is to `cd node_modules/js2coffee && npm install`. then you can quickly run _grunt-generate-configs_ via `node bin/generate_configs`.

@creynders thanks for opening an issue with `js2coffee`, https://github.com/js2coffee/js2coffee/issues/351 (obviously I found it after debugging myself :) )

**Comment by rstacruz** [comment]
thanks for the report... js2coffee should be fine now, just bump it to ^2.0.2.

**Comment by marocas** [comment]
@arminrosu how are did you manage to run ```bin/generate-configs``` from cmd-line?
I'm getting this error from cmd-line, i'm sorry if this is a dumb question, but im not a good programmer :)
``` ' generate_configs ' is not recognized as an internal or external command , operational or batch file program.` ```

**Comment by arminrosu** [comment]
@marocas you have to have node installed, then just `node bin/generate_configs` from the _grunt-generate-configs_ directory.

**Comment by creynders** [comment]
@rstacruz thanks for the swift response!!
@marocas ~~you can pass the bin file to node which will~~ Haha, @arminrosu beat me to it.

**Comment by creynders** [comment]
@marocas Could you let me know if this works for you? Published as v0.6.1

**Comment by marocas** [comment]
@creynders @arminrosu @rstacruz thank you guys for the help, im not a programmer, thats wht i asked for your help. After the update, all things when smooth like you explain on the README file.
Going for the next step, jit-grunt.

Thank you once again.
